### PR TITLE
CATTY-448 Display Message After Successful Upload

### PR DESCRIPTION
**Pull request by amelak9**
After a project has been uploaded successfully the user should see a message indicating that the upload finished successfully. Therefore an alert should be displayed at the landing page:

Title: "Upload successful"
Body: "Your project was uploaded to the app's sharing site where others can use and download it."
Buttons: "Show in Safari", "OK"
When the user clicks on "Show in Safari", the web browser should open and the project should be displayed on the share, i.e. using https://share.catrob.at/app/project/PROJECTID

The "Saved" notification which is currently appeared after a successful project upload should be removed.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Verify that the Jira ticket is in the status *Ready for Development*
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [ ] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#catty* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: src/Catty/ViewController/Upload/UploadViewController.swift [hunk 1]
```diff
@@ -424,7 +424,17 @@ class UploadViewController: UIViewController, UploadCategoryViewControllerDelega
...
-                                        project.rename(toProjectName: project.header.programName, andProjectId: projectId, andShowSaveNotification: true)
+                                        project.rename(toProjectName: project.header.programName, andProjectId: projectId, andShowSaveNotification: false)
+
+                                        AlertControllerBuilder.alert(title: kLocalizedProjectUploaded, message: kLocalizedProjectUploadedBody)
+                                            .addDefaultAction(title: kLocalizedSafariShow) {
+                                                if let projectURL = URL(string: kLocalizedBaseUrl + projectId) {
+                                                    UIApplication.shared.openURL(projectURL)
+                                                }
+                                            }
+                                            .addCancelAction(title: kLocalizedOK, handler: nil)
```
Comment (d4n3x) [inline src/Catty/ViewController/Upload/UploadViewController.swift:435]: If you make this also a default action the order is right.

File: src/Catty/ViewController/Upload/UploadViewController.swift [hunk 2]
```diff
@@ -424,7 +424,17 @@ class UploadViewController: UIViewController, UploadCategoryViewControllerDelega
                                 }
                                 DispatchQueue.main.async(execute: {
                                     if let projectId = projectId {
-                                        project.rename(toProjectName: project.header.programName, andProjectId: projectId, andShowSaveNotification: true)
+                                        project.rename(toProjectName: project.header.programName, andProjectId: projectId, andShowSaveNotification: false)
+
+                                        AlertControllerBuilder.alert(title: kLocalizedProjectUploaded, message: kLocalizedProjectUploadedBody)
+                                            .addDefaultAction(title: kLocalizedSafariShow) {
```
Comment (d4n3x) [inline src/Catty/ViewController/Upload/UploadViewController.swift:430]: If you add ",handler: {" after the kLocalizedSafariShow and move the if inside you could close with "})" and therefore the .addDefaultAction, .build and .showWithController would be aligned and easier to read.

**Review by d4n3x — DISMISSED** [review]

File: src/Catty/ViewController/Upload/UploadViewController.swift (hunk 2 above)
Comment (d4n3x) [reply src/Catty/ViewController/Upload/UploadViewController.swift:430]: This maybe in addition to the changes Michaels change request :D

**Review by m-herold — APPROVED** [review]
Thanks!

**Pull request merged**